app/helpers/settings_helper.rb

```diff
@@ -0,0 +1,20 @@
+module SettingsHelper
+  def formatted_setting_value(setting)
+    return setting unless filled_json_array?(setting)
+
+    elements = ''
+    values = JSON.parse(setting)
+    values.each do |value|
+      elements << content_tag(:a, value, class: 'ui label')
+    end
+    simple_format(elements)
+  end
+
+  def filled_json_array?(text)
+    JSON.parse(text).is_a?(Array) && JSON.parse(text).first.present?
+  rescue JSON::ParserError
+    false
+  rescue NoMethodError
```

Comment:
Why these `rescue` are needed? There must be a strong reason to catch an exception, since you basically suppress an error here.